packages/back-end/src/integrations/ClickHouse.ts

```diff
@@ -102,4 +102,7 @@ export default class ClickHouse extends SqlIntegration {
       );
     return `table_schema IN ('${this.params.database}')`;
   }
+  isAutoGeneratingFactTablesSupported() {
+    return true;
+  }
```

Comment:
There are other Clickhouse where their schema's are not going to match up, right?  So we want this to only be true for the growthbook-clickhouse datasource types.  I don't think we pass the type into the integration, but theoretically you could determine it if `this.params.host` matches up with `CLICKHOUSE_HOST`.  The env var right now has the protocol and the port included which originally seemed more useful, but in one other place we break it apart, so we could consider making the port as a separate env var.

Alternatively we could do a secondary check elsewhere that is using this to make sure that the datasource.type is `growthbook-clickhouse` but then we should at least comment it here because the name of the function would be misleading.

Reply:
This is also an old diff.